package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -195,6 +195,8 @@ def __init__(self,
                 (default is 50). Parameter in the Affinity Propagation for
                 clustering.
 
+            **kwargs being sent to scikit-learn.
```

Comment:
```suggestion
            **kwargs being sent to scikit-learn.
            **kwargs: Other keyword arguments are passed to :class:`sklearn.cluster.AffinityPropagation`.
```

The "sent to scikit-learn" was more casual phrasing. Something like the above is more suitable.

Reply:
Yeah this would be better.